state/allcollections.go

```diff
@@ -190,6 +190,12 @@ func allCollections() collectionSchema {
 			global: true,
 		},
 
+		// This collection holds information cached by autocert certificate
+		// acquisition.
+		autocertCacheC: {
+			global: true,
```

Comment:
I'm slightly surprised that I didn't get a panic trying to get the WriteCollection without this.